internal/serialization/serialization_test.go

```diff
@@ -68,6 +68,28 @@ func TestSerializationService_ToData(t *testing.T) {
 	}
 }
 
+func TestSerializationService_ToData_LittleEndianTrue(t *testing.T) {
+	var expected int32 = 100
+	c := &serialization.Config{}
+	c.LittleEndian = true
+	service, err := iserialization.NewService(c)
+	if err != nil {
+		t.Fatal(err)
+	}
+	data, err := service.ToData(expected)
+	if err != nil {
+		t.Fatal(err)
+	}
+	temp, err := service.ToObject(data)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
How about:
```
obj := it.MustValue(service.ToObject(data))
```

Reply:
Applied this comment to all tests at https://github.com/hazelcast/hazelcast-go-client/pull/912/commits/b9c72d900a0da806a1b7b419c59514656f2c5b84.